### DISCUSSION TITLE
Got my build together, using it for the first time tonight, so rad

### DISCUSSION THREAD

**Discussion by briwil** [FundamentalFrequency/.github#49] [Show and tell]
SSH'ing all my samples over is not so fun, but maybe I'll figure out a better (quicker) way.

![20220709_233120.jpg](https://user-images.githubusercontent.com/229978/178136988-daf0c4e3-1a70-4e90-9cd4-11ddfcafb1ee.jpg)

![20220709_233150.jpg](https://user-images.githubusercontent.com/229978/178136997-f15a61bc-b023-4613-8d50-57d035c6808e.jpg)

![20220710_011648.jpg](https://user-images.githubusercontent.com/229978/178137016-e74ec865-ffdd-4fb4-adaf-a86a5d2797a0.jpg)

![20220710_151118~2](https://user-images.githubusercontent.com/229978/178164149-92bf4475-0f3f-4762-89dc-02e690f3f7a8.jpg)
![20220710_151058~2](https://user-images.githubusercontent.com/229978/178164152-f0891866-2411-4e5b-8820-04766f72a41c.jpg)

Now that I have my wood panels in here, I kind of miss seeing the lemons on the board...

Also, does anyone know of a good way to print the keycaps? I see offers to do it on places like etsy, but it's expensive. Right now I just have some inkjet mailing labels that I printed on and used a Cricut machine to cut out, not so elegant.

**Comment by stonepreston** [comment]
The wood looks great. I just put all the samples in a folder on my host machine, then SSH all of them over with one command.

**Comment by Dylan-Grove** [comment]
I setup an SMB share on the pi so I can just map a drive to it from my PC. Then I can just drag and drop samples or even delete/rename them.

**Reply by briwil** [reply]
Oh nice, I'll try that.

**Reply by Dylan-Grove** [reply]
> Oh nice, I'll try that.

Yeah it works great. I love the little face plates you designed for the pi, would you mind sharing them? :)

**Reply by briwil** [reply]
I'm trying to remember what I posted here, I think it was the photo of the wood case, is that it? Because originally I also modeled a full enclosure that I 3D printed, but my 3D prints from my personal printer were a little wonky, things shrunk or got skewed... So I ended up converting that design to just a top and bottom plate that I got cut online (the wooden plates in the pic) as well as some side pieces I had cut at the same time (I went with metal for those, also seen in the pic). But I still had to 3D print some corner pieces to hold the metal sides and the top and bottom plates together (they're purple in my photo). I also printed a piece to cover the Pi's ports.

So sure, I can share the top, bottom, and side plate eps files, as well as the 3D file for the corner pieces. I'll need to gather them all, I'll post later today, once I figure out where on here to post such things.

The top and bottom pieces are pretty much the same as the original ones posted for this project, I just extended it out a bit to wrap around the screen completely.

I got my pieces cut at ponoko.com, there might be cheaper places but I had a credit with them already.

**Reply by Dylan-Grove** [reply]
Awesome! I like the port covers, really cleans up the device. Thanks so much for sharing, I look forward to it.

**Reply by briwil** [reply]
Here are my files (if there is somewhere on Github to post them, let me know)- https://www.dropbox.com/sh/ssmz6ky1df3f3cx/AABO_bKordvDWQcKZKWSYK6Ca?dl=0

It contains the eps files for the top, bottom, and sides that I used for my laser cuts. It also contains the 3D model I made (including raspi and screen) to help me plan everything out. There are also the side pieces I printed in the STL folder.
